app/models/ability.rb

```diff
@@ -1,9 +1,11 @@
 #
 # CanCan permissions
 #
+# rubocop:disable ClassLength
```

Comment:
Since this is meant for the whole file, it's better to add this as an exception to the rubocp.yml file.